Nos/Views/KnownFollowersView.swift

```diff
@@ -8,53 +8,48 @@ struct KnownFollowersView: View {
     @ObservedObject var author: Author
     @Environment(CurrentUser.self) private var currentUser
     
-    var followersRequest: FetchRequest<Follow>
-    var followersResult: FetchedResults<Follow> { followersRequest.wrappedValue }
+    /// The authors that the `source` author follows who also follow the `author`
+    @FetchRequest private var knownFollowers: FetchedResults<Author>
     
-    var followers: Followed {
-        followersResult.map { $0 }
-    }
-    
-    var knownFollowers: [Follow] {
-        followers.filter {
-            guard let source = $0.source else {
-                return false
-            }
-            return source.hasHumanFriendlyName == true &&
-            source != author &&
-            source != currentUser.author &&
-            currentUser.isFollowing(author: source)
-        }
+    /// The authors that will be featured with profile pictures and names
+    var displayedAuthors: ArraySlice<Author> {
+        knownFollowers
+            .filter { $0.hasHumanFriendlyName }
+            .filter { $0.profilePhotoURL != nil }
+            .prefix(2)
```

Comment:
In the Profile, the design always shows 3 avatars and 2 names. But in the cards it is not consistent like you say.
If you change it to 4, we need to test that more throughly as we never displayed more than 3 and the container would we wider. I don't think it will cause any harm, but still, we need to keep an eye on that.

Reply:
@martindsq good call. 4 didn't work lol. The photos overlapped in weird ways. I changed it back to 3.